sources/exception.asciidoc

```diff
@@ -157,29 +157,15 @@ Reasons for raising this exception include:
 * The extracted type is not marked with `@ExtractedValue`
 * The `@ExtractedValue` annotation is given more than once for one value extractor type
 
-[role="api"]
```

Comment:
I don't think it's a good idea. They are only examples. The tck-testable assertions should be present in the according section.